<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mv</class>
 <widget class="QWidget" name="mv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_mv_find" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_seachMV" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>28</height>
         </size>
        </property>
        <widget class="QLineEdit" name="lineEdit_searchMV">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>28</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>28</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#lineEdit_searchMV
{
	font-size: 12px;
	font-family: 微软雅黑;
	color: rgba(255, 255, 255, 1);
	background: rgba(255, 255, 255, 0.15);
	border-radius: 14px;
	padding-left:12px;
	padding-right:24px;
}</string>
         </property>
         <property name="placeholderText">
          <string>周杰伦 《搁浅》</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_searchMV">
         <property name="geometry">
          <rect>
           <x>176</x>
           <y>6</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/musicPlayer/image/musicPlayer/yyjy_ssxl_icon.png);</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>312</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea_mv">
     <property name="styleSheet">
      <string notr="true">QScrollArea { border:none; background: transparent; }
QScrollArea &gt; QWidget &gt; QWidget { background: transparent; }
QScrollArea &gt; QWidget &gt; QScrollBar { background: palette(base); }</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>340</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../MediaPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
